code/modules/surgery/organs/organ.dm

```diff
@@ -32,6 +32,13 @@ var/list/organ_cache = list()
 	var/sterile = 0 //can the organ be infected by germs?
 	var/tough = 0 //can organ be easily damaged?
 
+/obj/item/organ/Destroy()
+	processing_objects.Remove(src)
+	owner = null
```

Comment:
Technically this should always be `null` by the time it's `qdel`, unless you do something like qdel'ing it, directly, while it's still on a mob (which in and of itself shouldn't really happen).

That said, putting this here, just in case.